ios/Video/RCTVideo.m

```diff
@@ -1024,6 +1036,87 @@ - (void)setIgnoreSilentSwitch:(NSString *)ignoreSilentSwitch
   [self applyModifiers];
 }
 
+- (void)setTitle:(NSString *)title
+{
+  _title = title;
+}
+
+- (void)setArtist:(NSString *)artist
+{
+  _artist = artist;
+}
+
+- (void)setArtworkUrl:(NSString *)artworkUrl
+{
+    _artworkUrl = artworkUrl;
+}
+
+//updates existing nowPlaying info with image loaded from artworkUrl
+-(void)updateExistingNowPlayingDataArtwork:(NSString *)artworkUrl {
+    if( artworkUrl == nil ) {
```

Comment:
needs spacing lint.. I've debated running clang-format on this entire file but it would make determining the fork's dif against the source impossible.